osciwidget.cpp

```diff
@@ -1,19 +1,27 @@
 #include "osciwidget.h"
 
-// system includes
-#include <cmath>
-
 // Qt includes
 #include <QLineF>
 #include <QDebug>
 #include <QPainter>
 #include <QTimerEvent>
 
-OsciWidget::OsciWidget(QWidget *parent) :
-    QOpenGLWidget{parent},
-    m_redrawTimerId(startTimer(1000/m_fps))
+// system includes
+#include <cmath>
+
+qint32 framesForDuration(qint64 duration){
+    return qint32(44100 * duration / 1000000LL);
+}
```

Comment:
Why? It is in its own compilation unit anyways. Is there a way this can be visible somewhere else? I use anonymous namespaces only for files that will be included, but I do not know best practices for this

Reply:
Well you are right of course .. you could link against it if you reuse the name :)